### DISCUSSION TITLE
frontend module is not working

### DISCUSSION THREAD

**Discussion by prabii** [egovernments/Digit-Core#115] [Local Governance]
micro-ui  is not opening it failed and comprises with errors ....even though i installed  all the dependencies like node 14.20.0 and yarn 1.22.19 still i even follwed react ui customization still when they are running its perfectly fine .But when we are running its throwing error the exact file also not giving so pls reslove this micro_ui is failing in all versions from 2.7 to 2.9 and digit master oss i cloned all repos and i tried the same result

**Comment by subhashini-egov** [comment]
@jagankumar-egov can you please answer this query?

**Comment by jagankumar-egov** [comment]
Follow the steps mentioned as per 
https://core.digit.org/guides/developer-guide/ui-developer-guide/local-development-setup
https://core.digit.org/guides/developer-guide/ui-developer-guide/run-application
